src/Kunstmaan/SeoBundle/Controller/Admin/SettingsController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Kunstmaan\SeoBundle\Controller\Admin;
+
+use Kunstmaan\AdminBundle\Controller\BaseSettingsController;
+use Kunstmaan\SeoBundle\Entity\Robots;
+use Kunstmaan\SeoBundle\Form\RobotsType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+
+class SettingsController extends BaseSettingsController
+{
+    /**
+     * @Route(path="/admin/settings/robots", name="KunstmaanSeoBundle_settings_robots")
```

Comment:
Why put the full path here? It would better be in config (refer to https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/master/src/Kunstmaan/UserManagementBundle/Resources/config/routing.yml & the controllers there), so it can be overriden if anyone would want to...